docker/Dockerfile

```diff
@@ -1,14 +1,17 @@
-FROM registry.access.redhat.com/ubi8/python-39:latest
+ARG EL_VERSION=8
+ARG PYTHON_VERSION=39
+
+FROM registry.access.redhat.com/ubi${EL_VERSION}/python-${PYTHON_VERSION}:latest
 
 # We need to be root to setup the container image
 USER 0
 
 # Install required packages and setup ssh
-RUN yum install -y yum-utils openssh-server git sudo curl nc
+RUN yum install -y yum-utils openssh-server git sudo nc
```

Comment:
I think this can go. It was causing the following error because `curl-minimal` is already provided in el9:
```
Error: 
 Problem: problem with installed package curl-minimal-7.76.1-23.el9_2.2.x86_64
  - package curl-minimal-7.76.1-23.el9_2.2.x86_64 conflicts with curl provided by curl-7.76.1-23.el9_2.2.x86_64
  - conflicting requests
```